pkg/controller/admin/users_list.go

```diff
@@ -29,6 +29,12 @@ func (c *Controller) HandleUsersIndex() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		ctx := r.Context()
 
+		session := controller.SessionFromContext(ctx)
```

Comment:
Do we need to duplicate this for every handler? Middleware shouldn't let you past RequireAuth without session?

Reply:
We don't, and it's a lot of repetition, but it's specifically checking to make sure we don't panic if the middleware is not installed correctly.